lumigator/python/mzai/backend/backend/settings.py

```diff
@@ -10,7 +11,8 @@
 class BackendSettings(BaseSettings):
     # Backend
     DEPLOYMENT_TYPE: DeploymentType = DeploymentType.LOCAL
-    MAX_DATASET_SIZE: ByteSize = "50MB"
+    MAX_DATASET_SIZE_HUMAN_READABLE: Final[str] = "50MB"
```

Comment:
Is there a reason for this 50MB limit? If yes, can we add a comment here explaining it?

Reply:
I'm not sure, from the git history, it looks like it has always been set at ~50MB.

I just wanted to pull it out into a string so we could include it in the docs and prevent user frustration.

Reply:
@peteski22 I edited my comment, adding more context. Sorry for the latency I caused!